BranchInfo.props

```diff
@@ -12,7 +12,7 @@
     <IncludeBuildNumberInPackageVersion Condition="'$(StabilizePackageVersion)' != 'true'">true</IncludeBuildNumberInPackageVersion>
 
     <ReleaseSuffix>$(PreReleaseLabel)</ReleaseSuffix>
-    <ReleaseBrandSuffix>Preview</ReleaseBrandSuffix>
+    <ReleaseBrandSuffix>Preview4</ReleaseBrandSuffix>
```

Comment:
This is also missing a space.

Reply:
I wouldn't be opposed to using MSBuild property functions to lowercase and remove the space to keep brand and prerelease label in sync.